### PR TITLE
Use 32bit METIS in OpenFOAM-Extend 3.2

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/o/OpenFOAM-Extend/OpenFOAM-Extend-3.2-gimkl-2.11.5.eb [hunk 1]
```diff
@@ -25,7 +25,7 @@ patches = [
 dependencies = [
     ('hwloc', '1.11.1', '', ('GCC', '4.9.3')),
     ('ParMETIS', '4.0.3'),
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenFOAM-Extend/OpenFOAM-Extend-3.2-gimkl-2.11.5.eb]: not here?

Comment (wpoely86) [inline easybuild/easyconfigs/o/OpenFOAM-Extend/OpenFOAM-Extend-3.2-gimkl-2.11.5.eb]: nope, not needed as far as I can tell

**Comment by boegel** [comment]
merging this in to be included in EB v2.5.0, enhanced easyblock (https://github.com/hpcugent/easybuild-easyblocks/pull/784) will be merged post v2.5.0 release

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/366b3ca72d32b3c4b556 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/cce4aacff3164f7e1b1c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ad2f2bb282a48c526468 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/811b106a4eb1ba9c0d87 for a full test report.

**Comment by boegel** [comment]
@wpoely86: problem in updated easyblock?

**Comment by wpoely86** [comment]
Yes, the location differs between `3.1` and `3.2`. Changes pushed and retesting.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/3478559f7ca82f6e61ee for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2eb37c3ab832073fea63 for a full test report.

**Pull request merged**